modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/copy_folder.jsp

```diff
@@ -27,7 +27,7 @@ dlCopyFolderDisplayContext.setViewAttributes(liferayPortletResponse);
 		module="document_library/js/DDMFolderSelector"
 		props='<%=
 			HashMapBuilder.<String, Object>put(
-				"copyActionURL", dlCopyFolderDisplayContext.getActionURL()
+				"copyActionURL", dlCopyFolderDisplayContext.getCopyToURL()
 			).put(
 				"itemType", "folder"
```

Comment:
Instead of making up "type" identifiers, use `KBFolder.class.getSimpleName()` (== 'KBFolder'). Otherwise, we may end up using different type identifiers in different places.